docker/Dockerfile

```diff
@@ -37,9 +38,9 @@ RUN apt install -y \
         python3-pandas \
         python3-jinja2 \
         cython3 \
-        libwebsockets-dev \
-        python3-pprintpp \
-    && apt clean -y all
+        libwebsockets-dev
+RUN apt install python3-pprintpp; exit 0
```

Comment:
Could this be handled with this `apt install` options?
```suggestion
RUN apt install -m python3-pprintpp
```

Reply:
This was still throwing an error for me.  I played around with some different `apt install` options but couldn't find a more graceful way to do this

Reply:
** an error that stopped the docker build